dist/images/Makefile

```diff
@@ -28,6 +28,8 @@ fedora: bld
 	# docker push docker.io/ovnkube/ovn-daemonset-f:latest
 	./daemonset.sh --image=docker.io/ovnkube/ovn-daemonset-f:latest
 
+all: ubuntu centos fedora
```

Comment:
you should put this first; if you just do "`make`" with no arguments, it makes the first target in the Makefile